src/components/binding/binding-panel.js

```diff
@@ -215,13 +215,20 @@ genotet.BindingPanel.prototype.updateFileListAfterLoading = function() {
     this.selectGenes_[index] = select;
     select.val(track.fileName).trigger('change');
 
-    // Set track fileName
-    select.on('select2:select', function(event) {
-      var fileName = event.params.data.id;
-      this.signal('updateTrack', {
-        trackIndex: index,
-        fileName: fileName
-      });
-    }.bind(this));
+    // Set track fileName and highlight the track hovered in panel.
+    select
+      .on('select2:select', function(event) {
+        var fileName = event.params.data.id;
+        this.signal('updateTrack', {
+          trackIndex: index,
+          fileName: fileName
+        });
+      }.bind(this))
+      .on('select2:open', function(event) {
```

Comment:
Yes you're right. So I guess it's both hovering and clicking.

On Saturday, February 20, 2016, Liana Yang <notifications@github.com> wrote:

> In src/components/binding/binding-panel.js
> <https://github.com/ViDA-NYU/genotet/pull/115#discussion_r53557890>:
>
> > -      var fileName = event.params.data.id;
> > -      this.signal('updateTrack', {
> > -        trackIndex: index,
> > -        fileName: fileName
> > -      });
> > -    }.bind(this));
> > +    // Set track fileName and highlight the track hovered in panel.
> > +    select
> > +      .on('select2:select', function(event) {
> > +        var fileName = event.params.data.id;
> > +        this.signal('updateTrack', {
> > +          trackIndex: index,
> > +          fileName: fileName
> > +        });
> > +      }.bind(this))
> > +      .on('select2:open', function(event) {
>
> Does open event need trigger highlighting? I think it is necessary as well
> because the user may want to know which track they are going to change
> while they are selecting lists.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ViDA-NYU/genotet/pull/115/files#r53557890>.
>